src-java/flowhs-topology/flowhs-storm-topology/src/test/java/org/openkilda/wfm/topology/flowhs/mapper/HaFlowMapperTest.java

```diff
@@ -98,28 +106,31 @@ public void haFlowRequestMappingTest() {
         assertEquals(request.isStrictBandwidth(), result.isStrictBandwidth());
         assertEquals(request.getDescription(), result.getDescription());
         assertEquals(request.isAllocateProtectedPath(), result.isAllocateProtectedPath());
-        assertSharedEndpoint(request.getSharedEndpoint(), result.getSharedEndpoint());
+        assertEquals(request.getSharedEndpoint().getSwitchId(), result.getSharedSwitchId());
+        assertEquals(request.getSharedEndpoint().getPortNumber().intValue(), result.getSharedPort());
+        assertEquals(request.getSharedEndpoint().getOuterVlanId(), result.getSharedOuterVlan());
+        assertEquals(request.getSharedEndpoint().getInnerVlanId(), result.getSharedInnerVlan());
         // subflows must be mapped separately
         assertEquals(0, result.getSubFlows().size());
```

Comment:
To me, it is better to write those comments right in the assertion. It is displayed in the test results and less chances to forget the comment when moving lines